content/guides/references/error-messages.md

```diff
@@ -566,7 +566,8 @@ Since no record key was passed, Cypress checks for any environment variable with
 the name `CYPRESS_RECORD_KEY`. In this case, that was also not found.
 
 You can get your project's record key by locating it in your settings tab in the
-Cypress App or in the [Dashboard Service](https://on.cypress.io/dashboard).
+Cypress Test Runner or in the
```

Comment:
I think was already correct.

"...by locating it in your Settings tab of the Cypress app or...